pynsee/download/_download_store_file.py

```diff
@@ -49,18 +45,11 @@ def _download_store_file(id: str, update: bool):
                 potential values are: {suggestions}"
 
         raise ValueError(error_message)
+        
+    filename = tempfile.mkdtemp() + "tempfile"
```

Comment:
use ``os.path.join`` to make sure this will always work on all systems

Reply:
this commit d201b0795428266de9ac0b95e4f7b6179400f2a7 addresses the issue